Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM docker-upgrade.artifactory.build.upgrade.com/container-base:20180917-9
+LABEL maintainer=devops@upgrade.com
+
+USER root
+
+ENV I_KEY i_key
+ENV S_KEY s_key
+ENV HOST host
+ENV COLL_ENDPOINT endpoint
+ENV SCAN_INTERVAL_IN_SEC scan_interval_in_sec
+
+RUN amazon-linux-extras install python3 -y && pip3 install six botocore && yum clean all
```

Comment:
why not use our python base image then? 

https://github.com/Credify/python-base

Reply:
why is our python-base running a RC version? and not a release one? Might just need to rebuild the image to get python 3.7.1 ?